sql/ddl_api.sql

```diff
@@ -43,7 +43,7 @@ CREATE OR REPLACE FUNCTION  create_hypertable(
 
 -- Same functionality as create_hypertable, only must have a replication factor > 0 (defaults to 1)
 CREATE OR REPLACE FUNCTION  create_distributed_hypertable(
-    main_table              REGCLASS,
+    hypertable              REGCLASS,
```

Comment:
This would be `relation` since there is no hypertable at this point.
```suggestion
    relation              REGCLASS,
```